ibis/backends/sqlite/tests/conftest.py

```diff
@@ -69,43 +71,19 @@ def con(data_dir, tmp_path_factory, worker_id):
     return TestConf.load_data(data_dir, tmp_path_factory, worker_id).connection
 
 
-@pytest.fixture(scope="session")
-def dialect():
-    import sqlalchemy as sa
-
-    return sa.dialects.sqlite.dialect()
-
-
 @pytest.fixture(scope="session")
 def translate(dialect):
     from ibis.backends.sqlite import Backend
 
     context = Backend.compiler.make_context()
-    return lambda expr: str(
-        Backend.compiler.translator_class(expr, context)
-        .get_result()
-        .compile(dialect=dialect, compile_kwargs={"literal_binds": True})
-    )
-
-
-@pytest.fixture(scope="session")
-def sqla_compile(dialect):
-    return lambda expr: str(
-        expr.compile(dialect=dialect, compile_kwargs={"literal_binds": True})
-    )
+    return lambda expr: Backend.compiler.translator_class(expr, context).get_result()
```

Comment:
Should this be `ibis.to_sql(..., dialect="sqlite")`?

Reply:
IIRC I blindly copied that from a different backend. Looks fine, but we may want to cleanup the fixtures uniformly later.